apps/designer/app/canvas/shared/breakpoints.ts

```diff
@@ -33,7 +33,9 @@ const usePublishBreakpoints = () => {
 const useBreakpointChange = () => {
   useSubscribe("breakpointChange", (breakpoint) => {
     store.createTransaction([breakpointsContainer], (breakpoints) => {
-      const foundBreakpoint = breakpoints.find(({ id }) => id == breakpoint.id);
+      const foundBreakpoint = breakpoints.find(
+        ({ id }) => id === breakpoint.id
+      );
```

Comment:
why did this change?

Reply:
It is changed from `==` to `===`. Or you asking the styling?